ansible/checker.yml

```diff
@@ -0,0 +1,48 @@
+# This playbook deploys a simple stand alone checker for SAP/HANS.
+
+- hosts: all
+
+  tasks:
+    - name: include load_facts
+      import_role:
+        name: load_facts
+      tags:
+        - always
+
+    - name: include 1.1.1 remediate
+      import_role:
+        name: '1.1.1'
+      tags:
```

Comment:
Let's remove the `tags` by now, as we don't need them.
If we find that they are required in the future, we will see how we name and use them